lms/djangoapps/verify_student/tests/test_services.py

```diff
@@ -104,6 +104,11 @@ def test_user_status(self):
         status = IDVerificationService.user_status(user)
         self.assertEquals(status, {'status': 'must_reverify', 'error': '', 'should_display': False})
 
+        # test for when manual verification has been created
+        ManualVerification.objects.create(user=user, status='approved')
+        status = IDVerificationService.user_status(user)
+        self.assertEquals(status, {'status': 'approved', 'error': '', 'should_display': False})
```

Comment:
Can we use `self.assertDictEqual` here?

Reply:
I believe we can use that for a more robust assertion report, however, I am following the convention used for SSOVerification

Reply:
@irfanuddinahmad Feel free to update the `self.assertEquals` convention in `SSOVerification` and `SoftwareSecurePhotoVerification`. Might be worth doing if `self.assertDictEqual` will give a better assertion report.